openpgp/write_test.go

```diff
@@ -131,6 +132,57 @@ func TestNewEntity(t *testing.T) {
 	}
 }
 
+func TestEncryptWithCompression(t *testing.T) {
+	kring, _ := ReadKeyRing(readerFromHex(testKeys1And2PrivateHex))
+	passphrase := []byte("passphrase")
+	for _, entity := range kring {
+		if entity.PrivateKey != nil && entity.PrivateKey.Encrypted {
+			err := entity.PrivateKey.Decrypt(passphrase)
+			if err != nil {
+				t.Errorf("failed to decrypt key: %s", err)
+			}
+		}
+		for _, subkey := range entity.Subkeys {
+			if subkey.PrivateKey != nil && subkey.PrivateKey.Encrypted {
+				err := subkey.PrivateKey.Decrypt(passphrase)
+				if err != nil {
+					t.Errorf("failed to decrypt subkey: %s", err)
+				}
+			}
+		}
+	}
+
+	buf := new(bytes.Buffer)
+	var config = &packet.Config{
+		DefaultCompressionAlgo: packet.CompressionZLIB,
+		CompressionConfig:      &packet.CompressionConfig{-1},
+	}
+	w, err := Encrypt(buf, kring[:1], nil, nil /* no hints */, config)
+	if err != nil {
+		t.Errorf("error in Encrypt: %s", err)
```

Comment:
```suggestion
		t.Errorf("error in encrypting plaintext: %s", err)
```

To make it similar to the other errors